ui/src/components/toast/toast.styles.tsx

```diff
@@ -85,6 +85,7 @@ export abstract class ToastStyles {
     display: 'flex',
     flexDirection: 'column',
     gap: '$4',
+    zIndex: 4,

```

Comment:
We can standardize the z-index in the theme file, this would help keeping track of what comes on top of what in the app, through the settings. It could be variables like `$toast`, `$modal`, `$background` and so on.